aten/src/ATen/core/interned_strings.h

```diff
@@ -11,180 +11,181 @@
-  _(prim, Function)                \
-  _(prim, CreateObject)            \
-  _(prim, SetAttr)                 \
-  _(prim, GetAttr)                 \
-  _(prim, profile)                 \
-  _(prim, AddStatValue)            \
-  _(prim, TimePoint)               \
-  _(aten, append)                  \
-  _(aten, item)                    \
-  _(aten, format)                  \
-  _(aten, __not__)                 \
-  _(aten, __is__)                  \
-  _(aten, __isnot__)               \
-  _(aten, copy_)                   \
-  _(aten, t_)                      \
-  _(aten, addbmm_)                 \
-  _(aten, addcdiv_)                \
-  _(aten, addcmul_)                \
-  _(aten, addmv_)                  \
-  _(aten, addr_)                   \
-  _(aten, baddbmm_)                \
-  _(aten, ge_)                     \
-  _(aten, gt_)                     \
-  _(aten, le_)                     \
-  _(aten, lerp_)                   \
-  _(aten, lt_)                     \
-  _(aten, ne_)                     \
-  _(aten, transpose_)              \
-  _(aten, unsqueeze_)              \
-  _(aten, _set_item)               \
-  _(aten, manual_seed)             \
-  _(aten, set_)                    \
-  _(aten, index_put_)              \
-  _(aten, device)                  \
-  _(aten, hash)                    \
-  _(aten, len)                     \
-  _(aten, list)                    \
-  _(aten, wait)                    \
-  _(aten, save)                    \
-  _(aten, ord)                     \
-  _(prim, unchecked_unwrap_optional)\
-  FORALL_ATEN_BASE_SYMBOLS(_)      \
-  _(onnx, Add)                     \
-  _(onnx, Concat)                  \
-  _(onnx, Constant)                \
-  _(onnx, ConstantFill)            \
-  _(onnx, Div)                     \
-  _(onnx, GRU)                     \
-  _(onnx, Gather)                  \
-  _(onnx, Gemm)                    \
-  _(onnx, LSTM)                    \
-  _(onnx, Mul)                     \
-  _(onnx, Pow)                     \
-  _(onnx, RNN)                     \
-  _(onnx, Shape)                   \
-  _(onnx, Size)                    \
-  _(onnx, Slice)                   \
-  _(onnx, Squeeze)                 \
-  _(onnx, Sub)                     \
-  _(onnx, Transpose)               \
-  _(onnx, Unsqueeze)               \
-  _(onnx, Loop)                    \
-  _(onnx, If)                      \
-  _(onnx, Reshape)                 \
-  _(onnx, Equal)                   \
-  _(onnx, Greater)                 \
-  _(onnx, Less)                    \
-  _(onnx, Not)                     \
-  _(onnx, ATen)                    \
-  _(onnx, Split)                   \
-  _(onnx, ConstantOfShape)         \
-  _(onnx, Cast)         \
-  FORALL_ATTR_BASE_SYMBOLS(_)      \
-  _(attr, Subgraph)                \
-  _(attr, ReverseSubgraph)         \
-  _(attr, f_real_outputs)          \
-  _(attr, df_input_vjps)           \
-  _(attr, df_input_captured_inputs) \
+#define FORALL_NS_SYMBOLS(_)         \
```

Comment:
What happened here? This looks formatted fine before.

Reply:
yup; lemme revert formatting to this file.